frontend/src/pages/MaengelForm.js

```diff
@@ -166,16 +148,18 @@ export default function MaengelForm({ initialMode, title }) {
     // save contactLog and update mangel
     setLoading(true)
     setError()
-    putContactLog(token, id, contactLogger)
-      .then(setMangel)
+    const tempContactLogger = { ...contactLogger }
+    console.log(mangel.contactLoggerList)
+    putContactLog(token, id, tempContactLogger)
+      .then(() => getMangelById(token, id).then(setMangel))
```

Comment:
.then in .then block ist nicht so schön. du kannst den then einfach hinten anhängen. da ja getMangel wieder ein promise zurückgibt

.then(()=>getMangel(token,id))
.then(setMangel)

Reply:
👍